Assets/AppCenter/Plugins/AppCenterSDK/Core/UWP/AppCenterInternal.cs

```diff
@@ -122,7 +122,9 @@ public static void SetCustomProperties(object properties)
                                          string liveUpdateDeploymentKey,
                                          string liveUpdatePackageHash)
         {
-            //TODO once wrapper sdk exists for uwp, set it here
+            Prepare();
+            var wrapperSdk = new Microsoft.AppCenter.WrapperSdk(wrapperSdkVersion, wrapperSdkName, wrapperRuntimeVersion, liveUpdateReleaseLabel, liveUpdateDeploymentKey, liveUpdatePackageHash);
```

Comment:
I changed code to have wrapperSdkName first in the .net repo before merging.
```suggestion
            var wrapperSdk = new Microsoft.AppCenter.WrapperSdk(wrapperSdkName, wrapperSdkVersion, wrapperRuntimeVersion, liveUpdateReleaseLabel, liveUpdateDeploymentKey, liveUpdatePackageHash);
```

Reply:
Interesting to see unity was doing version first ah ah.